### DISCUSSION TITLE
Getting additional form config fields on the frontend

### DISCUSSION THREAD

**Discussion by dnrdigital** [statamic/cms#10606] [Q&A] [answered]
hi, I've been adding config files to my forms now that [this PR](https://github.com/statamic/cms/pull/8491) is in core. That's working great: i can add the fields, save the form and then my forms have a yaml that looks like:
```
success_message: "Here's a lovely success message!"
success_redirect: true
redirect_link: 'entry::6f....'
```
But I can't easily access them on the front end. Inside my {{ form:create }} tags, I want to do something like:

`{{ partial:components/notification type="success" content="{ success_message ? success_message : trans:strings.form_success }" }}`

That doesn't work, and I can't seem to get to anything in the form config. Do I need another step to expose this stuff on the frontend?

**Comment by ryanmitchell** [comment, marked as answer]
Currently those arent available to antlers - I've added a PR here to support that: https://github.com/statamic/cms/pull/10616

**Reply by dnrdigital** [reply]
ah awesome, that's great!
